### DISCUSSION TITLE
Which XDM should I use?

### DISCUSSION THREAD

**Discussion by J-HS-L** [subhra74/xdm#1070] [Q&A]
Please see the attached screenshot....

Ages back I did lots of digging into this, but now I can't recall why I downloaded & installed both of these!
Could some folks please explain **all** of the differences between them, & which are best for x/y reason(s)? 
For now, mainly just for use on Windows 10 device(s)... Thank you!

![Cap4454ture](https://github.com/subhra74/xdm/assets/49305869/b740e3a0-20a4-4938-a771-0a7b563fc0d6)

**Comment by J-HS-L** [comment]
@subhra74 ?

I had a look at [this site](https://xtremedownloadmanager.com/), but there's no explanation there, either.

Thanks again!

**Comment by armankuyucu** [comment]
Developer explains the difference between them here: https://github.com/subhra74/xdm/discussions/768
The main difference is that the new version is written using C# and the old version is written using Java.

**Reply by J-HS-L** [reply]
@armankuyucu 
Are you sure that's the only difference, or are there some feature differences etc?
That's a really informative thread, I will gradually absorb it all, thanks!
